docs/Project-Configuration/Project-Commands.md

```diff
@@ -600,6 +600,8 @@ Parameters:
     "github_token" and then set this field to \${github_token}.
     Evergreen will populate the expansion when it parses the project
     yaml.
+-   `is_oauth`: If a project token is provided and that token is an OAuth token and not a
```

Comment:
should we require this field to be set if a token is provided?

Reply:
Is there a way we could look at the token's prefixes and set this automatically ourselves?

Reply:
the default will be false and the default will be github apps so I don't think it should be required. 

I don't think there's a guaranteed way to parse it, especially not with future guarantees. There's currently only one project using an oauth token and I don't expect that to increase by much so I don't expect this to add a lot of overhead for users.